docs/pages/faq.mdx

```diff
@@ -149,6 +149,8 @@ Teleport provides security-critical support for the current and two previous rel
 
 | Release | Long Term Support | Release Date | Min tsh version |
 | - | - | - | - |
+| 6.2 | No | May 21th, 2021 | 3.1 |
+| 6.1 | No | April 9th, 2021 | 3.1 |
```

Comment:
I think we need to bump Min tsh version? cc @russjones

Reply:
I'm going with 4.3.x since it's the earliest I see here: https://goteleport.com/teleport/download/

Reply:
This is compared here:

https://github.com/gravitational/teleport/blob/b86a53611b28155d198db60fe36599bbcf1c7127/lib/utils/ver.go#L43

And appears to use:

https://github.com/gravitational/teleport/blob/b86a53611b28155d198db60fe36599bbcf1c7127/constants.go#L581

Reply:
We are planning on fixing our semvar https://github.com/gravitational/teleport/blob/master/rfd/0012-teleport-versioning.md . I would just bump to 4.3.x so it's at least a little newer